documentation/modules/ROOT/pages/podman-intro.adoc

```diff
@@ -646,4 +660,30 @@ include::partial$copy_visitor_info.adoc[]
 .`{guestbook-record-file}` restored
 image::visitor-info.png[]
 
-But what good are containers if we can never _persist_ changes we've made with them without all this stopping and committing them?  Let's look to the next section to see if there is something else that can be done. 
\ No newline at end of file
+But what good are containers if we can never _persist_ changes we've made with them without all this stopping and committing them?  Let's look to the next section to see if there is something else that can be done.
+
+[#automatic_container]
+== 9. Start Containers automatically
+
```

Comment:
@bfarr-rh I'm not sure this section is relevant for a container intro, especially if we don't need to use it in the creation of our website container.  It feels very targeted to linux sysadmins which could be our target audience but also might not. 
 Is there some other place this might make more sense (and where we can demonstrate it working?  participants can reboot their instances with `sudo shutdown -r now` if necessary)?  Maybe in one of the optional modules?